data_structures/src/mainnet_validations.rs

```diff
@@ -209,6 +209,32 @@ pub fn after_second_hard_fork(epoch: Epoch, environment: Environment) -> bool {
     epoch >= SECOND_HARD_FORK && Environment::Mainnet == environment
 }
 
+/// Allows to check the active Witnet Improvement Proposals
+#[derive(Clone, Debug)]
+pub struct ActiveWips {
+    pub active_wips: HashMap<String, Epoch>,
+    pub block_epoch: Epoch,
+    pub environment: Environment,
+}
```

Comment:
As a general note, there isn't necessarily a 1:1 relation between a "protocol improvement" and an "improvement proposal".

That is, the improvements from multiple WIP documents can be signaled and activated as a single "protocol improvement".

Therefore, using the WIP term in the code seems misleading to me, because e.g. you can't make your node oppose a WIP, because WIPs are opposed by written word and discussion—what you can do here is signal that your node is not ready to activate the protocol changes proposed by the WIP (which cannot exist at the TAPI level before they have been "socially" approved by he community)

I don't have a better name though, so I can't really demand this to be changed and I'd be actually OK to leave it as is.

Reply:
ActiveWpi? (Witnet Protocol Improven)

WPI vs WIP 🤯

Reply:
We could also add a "description" field that explain what wips are included in that wpi

Reply:
> WPI vs WIP

No.